arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/copy/ReturnDataCopySection.java

```diff
@@ -47,6 +48,7 @@ public ReturnDataCopySection(Hub hub) {
 
     // returnDataCopyException case
     if (returnDataCopyException) {
+      commonValues.setTracedException(TracedException.RETURN_DATA_COPY_FAULT);
```

Comment:
I'm thinking that if we do a check for `RETURNDATACOPY` and the presence of a `RETURN_DATA_COPY_FAULT` (which, BTW, we should rename to `RETURN_DATA_COPY_EXCEPTION`) first then we can include the COPY family in the the families where we have the high level check.